app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryStateActions.js

```diff
@@ -40,11 +42,20 @@ class CollapsedStoryStateActions extends React.Component {
   }
 
   needConfirmation(action) {
-    return action === storyTransitions.ACCEPT || action === storyTransitions.REJECT;
+    return action === storyTransitions.ACCEPT ||
+      action === storyTransitions.REJECT ||
+      action === storyTransitions.RELEASE;
+  }
+
+  getState(story) {
+    return isRelease(story.storyType) && !isAccepted(story)
+      ? status.RELEASE
+      : story.state;
   }
 
   render() {
     const { story, onUpdate } = this.props;
+    const state = this.getState(story);
```

Comment:
Avoid having the method `getState` in a class that `setState` does something waaaay different.